source/_integrations/zha.markdown

```diff
@@ -67,6 +67,22 @@ There is currently support for the following device types within Home Assistant:
 
 There is also support for grouping of lights, switches, and fans (i.e. support for commanding device groups as entities). At least two entities must be added to a group before the group entity is created. As well as support for binding/unbinding (i.e. bind a remote to a lightbulb or group).
 
+## Introduction
+
+ZHA integration follows the official Zigbee specification standards and uses one Zigbee Coordinator to set up and control a Zigbee network. Once ZHA has been installed and the Zigbee Coordinator is configured you will be able to directly join/pair Zigbee Router Device and Zigbee End Device products to that Zigbee network, regardless of the manufacturer and brand of that Zigbee-based product. Please see the respective sections below for compatible Zigbee radio modules hardware (physical radio adapter) and information on supported Zigbee devices.
+
+Before installing the integration you will need to buy and connect a Zigbee Coordinator adapter to the computer running the Home Assistant installation. The general recommendation is to buy a newer model of Zigbee Coordinator adapter hardware and to flash the latest firmware to it, the reason that is it will usually offer better interoperability with all functions of most Zigbee 3.0 compliant devices on the market.
+
+Supported devices are Zigbee Router Devices, which are normally mains-powered that act as Zigbee signal repeaters within the Zigbee network mesh to extend its range and improve coverage, and Zigbee End Devices, which will not act as Zigbee signal repeaters and are usually but not always battery-operated sensors.
+
+### Limitations
```

Comment:
I don't think limitation belong at the start of documentation. Know limitations are usually listed in their own chapter somewhere at the end.

For example, the fact you can only use a single coordinator, isn't relevant at this point of the documentation at all. Nor does it matter if your devices are already paired or not at this point (that part would better fit in a pairing paragraph).

Reply:
OK, I moved the Limitation into its own section as suggested, so it is now under Troubleshooting as I think the users should be aware of the limitations before they start troubleshooting.